commandapi-core/src/main/java/dev/jorel/commandapi/arguments/AbstractArgument.java

```diff
@@ -361,6 +361,10 @@ public void copyPermissionsAndRequirements(Argument argument) {
 		this.withPermission(argument.getArgumentPermission());
 	}
 
+	public String getAsArgumentString() {
```

Comment:
Nitpick: This method name is confusing. This wraps the argument's _node name_ in `<` and `>` characters, so I would expect this method's name to mention something along the lines of _node name_.

Reply:
Hmm, I was thinking about this in a different way. The help generation originally used the `RegisteredCommand#argsAsStr()` method which contents will be something along the lines of `nodeName:ClassName`.
When the help is generated, this split at the colon and the node name is wrapped in `<` and `>`.
Hence I came up with that name.
I don't think a name along the lines of *node name* makes much sense, or maybe I currently can't think of a nice name.
If I had to change this name, I guess I'd currently go with `getHelpString` or `getAsArgumentHelpString` but those aren't *that* great either.
I do agree that the current name isn't fitting now that you mention it and I could think about this again but I just don't have a good alternative at the moment.

Reply:
I think `getHelpString` works - it's the string representation of an argument for use in command help.